### DISCUSSION TITLE
New Omada Software Controller Released

### DISCUSSION THREAD

**Discussion by psychedelicu** [tteck/Proxmox#2569] [Help Needed] [answered]
Hi.

A new version of Omada Software controller was released today. How can i update it?

**Comment by tteck** [comment]
I don't see it on their website, It still shows v5.13.23 as the latest.
https://www.tp-link.com/us/support/download/omada-software-controller/#Controller_Software

**Reply by noing** [reply, marked as answer]
use the en site
https://www.tp-link.com/en/support/download/omada-software-controller/

latest_url=$(curl -fsSL "https://www.tp-link.com/en/support/download/omada-software-controller/" | grep -o 'https://.*x64.deb' | head -n1)

and it works.

Maybe you should curl both pages and check which is the latest version.

**Reply by tteck** [reply]
That's a good idea.

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/commit/05dc5371243fac3891ff14b1a67cecf03d85b061

**Comment by tteck** [comment]
For updating, it's as simple as reading the instructions.

![Screenshot 2024-02-27 4 59 52 AM](https://github.com/tteck/Proxmox/assets/21342670/99aeb847-cbb9-4bab-ad3a-ed3f4e482d54)

**Comment by psychedelicu** [comment]
I used this. https://www.reddit.com/r/TPLink_Omada/comments/rb3jjk/update_software_controller_with_proxmox_running/
I tried to update from the console, but it said that everything is up to date.

But thank you!

**Reply by tteck** [reply]
That's a 2 year old post.

**Comment by markuskonojacki** [comment]
Well there is a v5.13.30.8. But only for windows it seems.

![grafik](https://github.com/tteck/Proxmox/assets/657768/349781b3-0ee5-43f5-828c-f24363545f8b)

I was confused as well because the software says there is an update:
![grafik](https://github.com/tteck/Proxmox/assets/657768/d1a17d9f-09ea-4e01-b7e5-470716e8f1be)

**Reply by tteck** [reply]
It's surprising that they would show a Windows version on Linux.

**Comment by psychedelicu** [comment]
I've received an update notification here, in Omada Software Controller.
![image](https://github.com/tteck/Proxmox/assets/35727400/6a95e88e-421b-4910-9a9b-f24f591688f1)
